package.json

```diff
@@ -9,6 +9,7 @@
     "clean": "npm run clear-build-cache && lerna clean",
     "clear-build-cache": "rimraf ./packages/*/build/*",
     "copy-models": "node ./scripts/copyModels.js",
+    "lint": "tslint -c tslint.json './packages/*/src/*.ts' -t stylish || exit 0",
```

Comment:
Why is `|| exit 0` needed here?

Reply:
tslint returns status 2 if there are any linting errors, which will cause npm to error:

```
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! aws-sdk-js-v3@0.0.1 lint: `tslint -c tslint.json './packages/*/src/*.ts' -t stylish`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the aws-sdk-js-v3@0.0.1 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Reply:
The NPM error is expected here because the lint fails. 
I find the linting error is actually more than I expected and some of the validations are maybe too tight. How about we keep this PR open as we try to get rid of all the linting errors? We might need to turn back and update the config here.